languages/tree-sitter-stack-graphs-java/src/stack-graphs.tsg

```diff
@@ -570,27 +570,56 @@ attribute node_symbol = node            => symbol = (source-text node), source_n
   type: (_) @type
   declarator: (variable_declarator
     name: (_) @name
```

Comment:
You could remove `name` here, I think.

Reply:
Just checked - yup, the tests still pass, thanks for catching that we don't need this one anymore 😄 Pushed an update.